Worker/src/Unified/TaskSubmitterWorkerService.cs

```diff
@@ -144,6 +144,52 @@ public void ConfigureSessionService(ITaskHandler taskHandler)
     => SessionService = new SessionPollingService(LoggerFactory,
                                                   taskHandler);
 
+  /// <summary>
+  ///   The middleware triggers the invocation of this handler just after a Service Instance is started.
+  ///   The application developer must put any service initialization into this handler.
+  ///   Default implementation does nothing.
+  /// </summary>
+  /// <param name="serviceContext">
+  ///   Holds all information on the state of the service at the start of the execution.
+  /// </param>
+  public void OnCreateService(ServiceContext serviceContext)
```

Comment:
Is there any test from main with this new implementation of Callback ?

Reply:
@alouvion-aneo ?

Reply:
Hello yes https://github.com/aneoconsulting/ArmoniK.Extensions.Csharp/blob/main/Tests/EndToEnd.Tests/Tests/CheckSubtaskingTreeUnifiedApi/SubtaskingTreeUnifiedApiWorker.cs

Reply:
Those functions were removed in be47525903ab37efcbe00dbf22a6e94dc41fb7e7 , and since https://github.com/aneoconsulting/ArmoniK.Extensions.Csharp/blob/next/Tests/ArmoniK.EndToEndTests/ArmoniK.EndToEndTests.Worker/Tests/CheckSubtaskingTreeUnifiedApi/SubtaskingTreeUnifiedApiWorker.cs is the direct replacement of the file given above, we can safely remove them again